pkgs/games/openarena/default.nix

```diff
@@ -10,6 +10,13 @@ stdenv.mkDerivation {
     sha256 = "0jmc1cmdz1rcvqc9ilzib1kilpwap6v0d331l6q53wsibdzsz3ss";
   };
 
+  # https://openarena.fandom.com/wiki/OACMP/Volume_1
+  communityMaps = fetchurl {
```

Comment:
Should this be an optional part of the openarena package?

Reply:
It should definitely be optional but I think it can be enabled by default.

Reply:
I don't know how to make it optional. `openarena.withMaps [ communityMaps ]` would be a nice pattern for this?

Reply:
You could have a `communityMaps ? true` as parameter and use `lib.optionalString communityMaps` I think, but maybe it would be better to convert the maps to another derivation and add a parameter `withMaps` like you said.

Would be cleaner and more flexible that way.